.gitignore

```diff
@@ -10,3 +10,6 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+# Explorer web app static files
+forc/www
```

Comment:
`www` should not be downloaded into the `forc` directory specifically, even for testing.
```suggestion
```